WooCommerce/Classes/ViewModels/Order Details/OrderDetailsResultsControllers.swift

```diff
@@ -30,6 +30,15 @@ final class OrderDetailsResultsControllers {
         return ResultsController<StorageProduct>(storageManager: storageManager, matching: predicate, sortedBy: [descriptor])
     }()
 
+    /// ProductVariation ResultsController.
+    ///
+    private lazy var productVariationResultsController: ResultsController<StorageProductVariation> = {
+        let variationIDs = order.items.map { $0.variationID }.filter { $0 != 0 }
```

Comment:
I was worried about this. I questioned myself “what if `order.items` changes?“. But I suppose it's fine because `self.order` is readonly. 🤷 

Nitpick! 😄 

```suggestion
        let variationIDs = order.items.map(\.variationID).filter { $0 != 0 }
```

Reply:
nice! updated in https://github.com/woocommerce/woocommerce-ios/pull/3197/commits/7dbe1f3ebf33f3e571285748bc76007225b1618f